docs/jwst/background/reference_files.rst

```diff
@@ -7,7 +7,7 @@ WAVELENGTHRANGE reference file is used in the process of determining the
 locations of source spectra in the image, and conversely the image areas
 that contain only background signal.
 
-.. include:: ../references_general/wfssbkg_reffile.rst
+.. include:: ../references_general/wfssbkg_reffile.inc
 
-.. include:: ../references_general/wavelengthrange_reffile.rst
+:ref: `WAVELENGTHRANGE <../includes/wavelengthrange_reffile.rst>`
```

Comment:
I can move them all to `.inc`. I only moved those files which produced warnings but agree for consistency it's best to move them all. In general, if the file has an extension `.rst` it is parsed and if it is in an `include` directive a `Warning` about a duplicate label s issued. 

There were 2 or 3 cases ( `wavevlengthrange`, `gain` and perhaps one more) where the `inc` files also produced warnings (I'm not sure why). In those cases the file was included in more than one step. The only way I could remove the warning is by referencing instead of including them. I should look some more for a different solution in my free time.